rpcs3/Emu/Cell/Modules/cellSpurs.cpp

```diff
@@ -4759,9 +4766,12 @@ s32 cellSpursJobGuardReset(vm::ptr<CellSpursJobGuard> jobGuard)
 	if (!jobGuard.aligned())
 		return CELL_SPURS_JOB_ERROR_ALIGN;
 
-	auto [res, rtime] = vm::reservation_lock(jobGuard.addr(), 128, vm::dma_lockb);
-	jobGuard->ncount0 = jobGuard->ncount1;
-	res.release(rtime + 128);
+	vm::reservation_light_op(jobGuard->ncount0, [&](atomic_be_t<u32>& ncount0)
+	{
+		// TODO: is it correct for light op?
+		ncount0 = jobGuard->ncount1;
```

Comment:
ncount1 is supposed to be some kind of constant value to save the original count's value, realhw only uses stw so its safe to use vm::reservation_light_op here.